ios/RNNetworkInfo.m

```diff
@@ -148,6 +149,22 @@ @implementation RNNetworkInfo
     }
 }
 
+RCT_EXPORT_METHOD(getGatewayIPAddress:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+    @try{
+	NSString *ipString = nil;
+	struct in_addr gatewayaddr;
+	int r = getdefaultgateway(&(gatewayaddr.s_addr));
+	if(r >= 0) {
+        	ipString = [NSString stringWithFormat: @"%s",inet_ntoa(gatewayaddr)];
+    	}
+        resolve(gatewayaddr);
```

Comment:
Don't we want to resolve the `ipString`?

Reply:
Good catch!